config/locales/en.yml

```diff
@@ -315,6 +315,11 @@ en:
       popular_links_title: Most read
       related_links_title: Related guides
 
+    inline_sign_up:
```

Comment:
Yes, good point however this paragraph is very article specific and just above the sharing icons so I think it's better suited above the article.sharing locale section.

n.b. I'm sure the whole locale files file could be refactored to be better organised site-wide but I don't want to open a can of worms and delay getting this released. Perhaps we can create a ticket to organise this?